src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1542,7 +1542,7 @@ private void StartAutoEvoPrediction()
     {
         biome ??= Editor.CurrentPatch.Biome;
 
-        var energyBalance = ProcessSystem.ComputeEnergyBalance(organelles, biome, membrane, true,
+        var energyBalance = ProcessSystem.ComputeEnergyBalance(organelles, biome, membrane, true, !HasNucleus,
```

Comment:
Considering the parameters of this method, the nucleus should be detected from the `organelles` readonly collection.

Reply:
I'm sorry, could you elaborate on what I need to do here? I'm currently using the bool as called on line 423.
`  public bool HasNucleus => PlacedUniqueOrganelles.Any(d => d == nucleus);`

Reply:
You need to do the same thing here, basically. As that variable needs to relate to the inputs of this method and not the current editor stage.

Something like:
```
    private void CalculateEnergyAndCompoundBalance(IReadOnlyCollection<OrganelleTemplate> organelles,
        MembraneType membrane, BiomeConditions? biome = null)
    {
        biome ??= Editor.CurrentPatch.Biome;

        bool hasNucleus = organelles.Any(d => d == nucleus);

        var energyBalance = ProcessSystem.ComputeEnergyBalance(organelles, biome, membrane, true, !hasNucleus,
```